lib/WebpackOptionsDefaulter.js

```diff
@@ -81,7 +88,7 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			return options.output.library || "";
 		});
 		this.set("output.libraryTarget", "var");
-		this.set("output.path", process.cwd());
+		this.set("output.path", path.join(process.cwd(), "dist"));
```

Comment:
This would mean

``` js
output: {
  filename: "bundle.js"
}
```

will write to `./dist/bundle.js`.

Is this expected?

Reply:
Yes definitely. Although I don't have stats yet to back this up. I'd say this is a pretty common pattern from seeing eall the diff examples etc.